tiup/tiup-command-mirror-rotate.md

```diff
@@ -0,0 +1,60 @@
+---
+title: tiup mirror rotate
+---
+
+# tiup mirror rotate
+
+`root.json` is an important file in a TiUP mirror. It stores the public keys needed for the entire system and is the basis of the chain of trust in TiUP. It mainly contains the following parts:
+
+- Signatures of mirror administrators. For the official mirror, there are five signatures. For an initialized mirror, there are three signatures by default.
+- The public keys used to verify the following files:
+    - root.json
+    - index.json
+    - snapshot.json
+    - timestamp.json
+- Expiration date of `root.json`. For the official mirror, the expiration date is one year later than the creation date of `root.json`.
+
+For detailed description of TiUP mirror, see [TiUP Mirror Reference](/tiup/tiup-mirror-reference.md).
+
+You need to update `root.json` in the following cases:
+
+- Replace the key of the mirror.
+- Update the expiration date of certificate files.
+
+After the content of `root.json` is updated, the file must be re-signed by all administrators; otherwise, the client rejects the file. The update process is as follows:
+
+1. Update the content of `root.json`.
```

Comment:
@lucklove 
```suggestion
1. tiup-server updates the content of `root.json`.
```

Reply:
1. The user (client) updates the content of `root.json`